src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/EnumWithFlagsAttributeTests.cs

```diff
@@ -90,11 +97,11 @@ public enum HexFlagsEnumClass
 
             // Verify no CA1027: Mark enums with FlagsAttribute
             string codeWithoutFlags = GetCSharpCode_EnumWithFlagsAttributes(code, hasFlags: false);
-            VerifyCSharp(codeWithoutFlags);
+            await VerifyCS.VerifyAnalyzerAsync(codeWithoutFlags);
 
             // Verify no CA2217: Do not mark enums with FlagsAttribute
             string codeWithFlags = GetCSharpCode_EnumWithFlagsAttributes(code, hasFlags: true);
-            VerifyCSharp(codeWithFlags);
+            await VerifyCS.VerifyAnalyzerAsync(codeWithFlags);
         }
 
         [Fact]
```

Comment:
💭 Why was this one not converted?

Reply:
As I have tried to explain in the PR message, I have some trouble with all tests using scope. They are all failing and I didn't yet had time to debug through and understand what is the issue.

Reply:
Thanks! We can leave them for now and I can look later. A big step towards the final goal means less work to eventually get there. 👍